runtime/oti/vmargs_api.h

```diff
@@ -55,6 +55,13 @@
 #define ENV_LD_LIB_PATH "LD_LIBRARY_PATH"
 #endif /* defined(OSX) */
 
+/**
+ * Walks the JavaLauncherOptionTable to find the entry for a given argument.
+ * The entries in this table represent arguments that are generated by the Java Launcher in OpenJDK.
```

Comment:
also describe parameters and returns, see `addOptionsDefaultFile` for an example